data/campaigns/Delfadors_Memoirs/scenarios/18_The_Portal_of_Doom.cfg

```diff
@@ -344,7 +346,7 @@
                 variable=unit
                 x,y=recall,recall
             [/unstore_unit]
-            {CLEAR_VARIABLE temp}
+            #   {CLEAR_VARIABLE temp}
```

Comment:
I did that because it goes with the commented-out code just out of sight above the diff.